src/site/content/en/blog/coop-coep/index.md

```diff
@@ -204,11 +204,11 @@ terminate the origin trial until this issue is safely resolved.
 
 {% endAside %}
 
-A browsing context group is a group of tabs, windows or iframes which share the
-same context. For example, if a website (`https://a.example`) opens a popup
-window (`https://b.example`), the opener window and the popup window share the
-same browsing context and they have access to each other via DOM APIs such as
-`window.opener`.
+A browsing context group is a group of tabs, windows, iframes or worker scripts
```

Comment:
I added an explanation of "browsing context group" because the term is used in other places in this document. I think precise understanding of what is browsing context group is unnecessary or distracting for general readers. I'd like to phrase this in a way it is not too detailed, but not telling a lie.

Do you have any suggestions?

Reply:
Maybe you can just say.

A browsing context group is a set of window that can reference each other. For instance, the top-level document and its child documents embedded via <iframe>, or popup opened using window.open.

Reply:
Can I say:

A browsing context group is a set of window that can reference each other. For instance, the top-level document and its child documents embedded via <iframe>, **worker scripts spawned within the document** or popup opened using window.open.

Reply:
I don't think so ;-)

DedicatedWorker can interact only with its creator, so I won't feel bad saying they "belong" to the same group as their creator, in an extended sense. However, SharedWorker and ServiceWorker can interact with every same-origin document, no matter their browsing context group. So, I would feel bad saying a SharedWorker/ServiceWorker is associated with a defined browsing context group.

Per the specification, I don't believe we can say a Worker is a "browsing context". They talk only about a window/document like object.

However, even if we don't really define browsing context group for workers, there are several COEP rules to know about:
- If a document has COEP, it's DedicatedWorker must have COEP.
- If a document has COEP, its SharedWorker must have COEP.
- ServiceWorker are free to live independently. However, avoiding a mismatch of COEP matter, for process allocation.
- To avoid leaking resources requested without COEP toward a context with COEP, we make responses sent using CacheStorage to execute the CORP check again on the response with the outgoing context, like they would have done during a network fetch.

Reply:
I will omit the `worker scripts spawned within the document` and use your suggested phrase as is.

```
A browsing context group is a set of window that can reference each other. For instance, the top-level document and its child documents embedded via <iframe>, or popup opened using window.open.
```

Could you rephrase this for me to better understand what it means?

```
ServiceWorker are free to live independently. However, avoiding a mismatch of COEP matter, for process allocation.
```

I wonder how to use ServiceWorker on cross-origin isolated environment is another tip developers might look for.

Reply:
We should ask developers to add COEP in both the Document and its ServiceWorker There are two reasons:
- The ServiceWorker process will be allowed to stay in the same process as their document. It saves resources and improves performances.
- A resource successfully received by the SW will always be allowed to be transferred to the document via the CacheStorage.

Contrary to DedicatedWorker, it is possible to use a ServiceWorker with a mismatching COEP. This is somehow a problem, because developers might not understand by themselves they should add COEP in ServiceWorker.

Reply:
Sounds like more complex than I have expected.
I'll resolve this issue for now just to cover Web Workers and will add Service Worker section later on.

Reply:
SGTM!